src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -267,7 +268,8 @@
           (cond-> {:dispatch [:navigate-back]}
             (= method constants/web3-send-transaction)
             (assoc :dispatch-later [{:ms 400 :dispatch [:navigate-to-modal :wallet-transaction-sent-modal]}]))
-          {:dispatch [:navigation-replace :wallet-transaction-sent]}))))))
+          {:dispatch-n [[:navigation-replace :wallet-transaction-sent]
+                        [:execute-stored-command]]}))))))
```

Comment:
Is there any particular reason why this has to be additional dispatch (and event) instead of just calling the `:events.commands/execute-stored-command` function and incorporating its effects ?

Reply:
probably not, except mixpanel related stuff (because we have `[:navigation-replace :wallet-transaction-sent]` trigger). Will check this

Reply:
should be fixed now